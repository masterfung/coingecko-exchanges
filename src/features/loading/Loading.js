import { Spin } from "antd";
import Title from "antd/lib/typography/Title";

import "./Loading.scss";

const Loading = () => {
  return (
    <div className="loading-container">
      <Spin className="spinner" />
      <Title>Loading...</Title>
    </div>
  )
}

export default Loading;