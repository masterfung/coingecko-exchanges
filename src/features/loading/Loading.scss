.loading-container {
  width: 300px;
  text-align: center;
  background-color: white;
  position: absolute;
  left: 45%;
  top: 50%;
  color: black;

  .spinner {
    font-size: 30px;
  }
}