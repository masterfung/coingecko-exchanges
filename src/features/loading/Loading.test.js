import { shallow } from "enzyme";
import Loading from "./Loading";

describe("Loading component", () => {
  it("should load and render properly", () => {
    expect(shallow(<Loading />)).toMatchSnapshot();
  });
})