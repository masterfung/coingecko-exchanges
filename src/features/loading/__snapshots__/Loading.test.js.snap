// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component should load and render properly 1`] = `ShallowWrapper {}`;
